README.md

```diff
@@ -0,0 +1,40 @@
+# Workspace client
+
+This is a client for workspace REST API and workspace master JSON-RPC API interactions.
+
+## Examples
+
+### REST API client
+
+```typescript
+import { Rest } from 'workspace-client';
+
+const restApiClient = Rest.getRestApi();
+// get list of workspaces
+const promise = restApiClient.getAll();
+promise.then((workspaces) => {
+    // process workspaces here
+});
+```
+
+### JSON-RPC API client
+
+```typescript
+import { JsonRpc } from 'workspace-client';
+
+const entryPoint = '/api/workspace';
+const masterApiClient = JsonRpc.getMasterApiClient(entryPoint);
+const connectionPromise = masterApiClient.connect(entryPoint);
+// get session ID
+connectionPromise.then(() => {
+    const clientId = masterApiClient.getClientId();
+});
+const statusChangeHandler = message => {
+    const status = message.status;
+};
+// subscribe to workspace status changes
+masterApiClient.subscribeWorkspaceStatus('workspace-id', statusChangeHandler);
```

Comment:
as we're on a workspace client, we could use only subscribe() for method name ? and provide type of subcription on method

subscribe(STATUS, workspaceId, () => {})

also, in the callback, can we get the event type ?

so for example I can with a single method subscribe to all events on a workspace

subscribe(ALL, workspaceID, () => {}}  and in the handler, I can do something different depending on the event

Reply:
I left method names as they were in Che Dashboard on purpose, I wanted to change Dashboard as few as possible. But later we could provide the only `subscribe` method as you proposed.